source/reference/minio-mc/mc-version.rst

```diff
@@ -60,6 +60,18 @@ Parameters
    Mutually exclusive with :mc-cmd:`~mc version suspend` and
    :mc-cmd:`~mc version info`
 
+.. mc-cmd:: --exclude-folders
```

Comment:
Is there a usual way to order parameters in the reference?

I put these two directly under `enable` because they are children, despite it not being obvious.

Examples would help (oops forgot to add) but is there more generally a better way? 

Besides reworking the entire page to get the parent/child relationships correct, as we discussed in slack.

Reply:
Parameters are supposed to go:
- Required params, alphabetically
   With the `:required:` class.
- Optional params, alphabetically
   With the `:optional:` class

I think we are still pretty inconsistent with this at the moment, but that's the order we're trying to get to.

Reply:
Sorted alpha within _required_ or _optional_.`--` is ignored.